### PR TITLE
Request inclusion of Tachyons.css

### PR DESCRIPTION
**Pull request by MaxMillion**

### CODE REVIEW

File: files/tachyons/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Adam Morse"
+github = "https://github.com/mrmrs/tachyons"
+homepage = "http://tachyons.io"
+description = "Fully responsive, performance first css patterns"
+mainfile = "tachyons.css"
```
Comment (jimaek) [inline files/tachyons/info.ini]: tachyons.min.css

File: files/tachyons/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "tachyons",
+  "repo": "mrmrs/tachyons",
+  "files": {
+    "include": ["css/tachyons.css","css/tachyons.min.css"]
```
Comment (jimaek) [inline files/tachyons/update.json]: I dont think the unminified file is needed in production. You can just leave the min

**Comment by MaxMillion** [comment]
Revised.

File: files/tachyons/info.ini (hunk 1 above)
Comment (jimaek) [inline files/tachyons/info.ini]: You forgot this.

Comment (MaxMillion) [inline files/tachyons/info.ini]: Thank you. Revised.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**